source-checker.pl

```diff
@@ -200,7 +200,13 @@ BEGIN
     if (%patches) {
         # parsing changes
         for my $changes (@changes) {
-            my $diff = diff "$old/$changes", "$dir/$changes";
+            my $diff = "";
```

Comment:
That one is probably superfluous now - as there is no way we will not do a diff in the end... but that's just nagging around